libs/designer-ui/src/lib/templates/templatesPanelFooter.tsx

```diff
@@ -0,0 +1,32 @@
+import { DefaultButton, PrimaryButton } from '@fluentui/react';
+import type { ReactNode } from 'react';
+
+export interface TemplatePanelFooterProps {
+  primaryButtonText: string | ReactNode;
+  primaryButtonOnClick: () => void | Promise<void>;
+  primaryButtonDisabled?: boolean;
+  onClose: () => void;
+}
+
+export const TemplatesPanelFooter = ({
+  primaryButtonText,
+  primaryButtonDisabled,
+  primaryButtonOnClick,
+  onClose,
+}: TemplatePanelFooterProps) => {
+  return (
+    <div className="msla-templates-panel-footer">
+      <PrimaryButton data-testid={'template-primary-button'} onClick={primaryButtonOnClick} disabled={primaryButtonDisabled}>
```

Comment:
nit: I would probably make the testid something more descriptive and use the 'data-automation-id' because that's what we've solidfied on for playwright. 
```suggestion
      <PrimaryButton data-automation-id={'ttemplate-footer-primary-button'} onClick={primaryButtonOnClick} disabled={primaryButtonDisabled}>
```